src/ldp/AuthenticatedLdpHandler.ts

```diff
@@ -13,6 +14,7 @@ import type { Operation } from './operations/Operation';
 import type { OperationHandler } from './operations/OperationHandler';
 import type { PermissionSet } from './permissions/PermissionSet';
 import type { PermissionsExtractor } from './permissions/PermissionsExtractor';
+import isNativeError = types.isNativeError;
```

Comment:
Minor nit: maybe we want to re-expose `isNativeError` from on of our own utilities, such that we can do a one-line `import { isNativeError } from '...'`